doc/source/intensity-map-script.rst

```diff
@@ -12,11 +12,11 @@ constants.
 For example, to plot a coherent neutron-weighted intensity map from CASTEP
 force constants along a recommended q-point path, one could run::
 
-   euphonic-intensity-map --weights coherent --energy-broadening 1.5 quartz.castep_bin
+   euphonic-intensity-map --weighting coherent --energy-broadening 1.5 quartz.castep_bin
```

Comment:
I notice that some seemingly-random words are highlighted in these shell code blocks, presumably because sphinx defaults to Python syntax highlighting. It looks like we could change the default language for certain files (e.g. these command-line-scripts pages) with `.. highlight:: bash`: https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Codeblocks.html#highlight-directive

Not really specific to PDOS so could be a separate PR.

Reply:
Opening a separate Issue for this.

Reply:
Thanks, #178